rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverterSpec.groovy

```diff
@@ -99,7 +106,7 @@ class PackageNameConverterSpec extends Specification {
         parsedRpmPackageName)
 
     then:
-      parsedDebPackageName == parsedRpmPackageName
+      parsedDebPackageName != parsedRpmPackageName
```

Comment:
Seems like we could compare each of the parts here to show that everything matches except for the `qualifiedName`.